src/cli/commands/config/index.ts

```diff
@@ -0,0 +1,25 @@
+import { Testplane } from "../../../testplane";
+import { CliCommands } from "../../constants";
+import logger from "../../../utils/logger";
+
+const { CONFIG: commandName } = CliCommands;
+
+export const registerCmd = (cliTool: typeof commander, testplane: Testplane): void => {
+    cliTool
+        .command(commandName)
+        .description("Lists all browsers from the config")
```

Comment:
did you forget to change the description?

Reply:
"Outputs Testplane config (including default and overriden by environment variables values)"